ipinfo/handler_async.py

```diff
@@ -347,3 +347,69 @@ def _ensure_aiohttp_ready(self):
 
         timeout = aiohttp.ClientTimeout(total=self.request_options["timeout"])
         self.httpsess = aiohttp.ClientSession(timeout=timeout)
+
+    async def getIterativeBatchDetails(
+        self,
+        ip_addresses,
+        batch_size=None,
+        raise_on_fail=True,
+    ):
```

Comment:
Is this code handling everything the same way the other `getBatchDetails` was? It looks like it might be missing stuff at a glance

Reply:
yes cache check was missing in one, it is now added.